tools/packaging/packaging.psm1

```diff
@@ -448,6 +448,12 @@ function New-PSSignedBuildZip
         Copy-Item -Path $_ -Destination $destination -force
     }
 
+    # Remove `signed` folder in buildpath now that signed binaries are copied
+    if (Test-Path $BuildPath\signed)
+    {
```

Comment:
Maybe put the brace on the same line as this seems to be the most common style? I have opened PR #5529 to help with the style in the future.

Reply:
I noticed both styles in this file, but it seems that for `if` statements, at least near the code I added, the braces were on a new line